utils/rest_api/create_task.py

```diff
@@ -0,0 +1,259 @@
+    print_hrule("END: Get Job Id")
+    if job_id_resp.status_code != 200:
+        print("CANNOT GET JOB ID, status code =", job_id_resp.status_code)
+        pprint(job_id_resp.json())
+        return res
+    job_id_json = job_id_resp.json()
+    pprint(job_id_json)
+    assert "segments" in job_id_json
+    segments = list(job_id_json["segments"])
+    assert segments
+    assert len(segments) == 1
+    assert "jobs" in segments[0]
+    jobs = segments[0]["jobs"]
+    assert len(jobs) == 1
+    job_id = jobs[0]["id"]
+    url_for_job = cvat_uri + "/?id={}".format(job_id)
+    print("url_for_job =", url_for_job)
+    res["url_for_job"] = url_for_job
+    res["status"] = True
+    return res
+
+def print_table_for_jira(list_all_done_res):
+    print_hrule("CURRENT RESULTS:")
+    for res in list_all_done_res:
+        assert res["status"]
+        print("|{}|[{}]|".format(res["full_relative_path"], res["url_for_job"]))
+    print_hrule("END OF CURRENT RESULTS")
+
+def main():
+    epilog_help_string = ("""
+    Please, note that argument path_json_labels_descr should point to a json file that describes labels for a task.
+    Example 1:
+        [
+            {
+                "name": "obj",
+                "attributes": []
+            }
+        ]
+    -- this will create for the task one bbox property with name 'obj' without attributes,
+    it is equivalent to CVAT label string 'obj'
+
+    Example 2:
+        [
+            {
+                "name": "obj",
+                "attributes": [
+                                   {
+                                      "name": "class",
+                                      "mutable": false,
+                                      "input_type": "text",
+                                      "default_value": "",
+                                      "values": [""]
+                                   }
+                ]
+            }
+        ]
+    -- this will create for the task one bbox property with name 'obj' with one non-mutable text attribute 'class',
+    it is equivalent to CVAT label string'obj @text=class:""'
+    """)
+
+    parser = ArgumentParser(description = "Script that receives list of videos and to creates CVAT tasks (one for the video), "
+            "upload the video to the task from the share, and print the table of created tasks that may be inserted into a JIRA ticket.",
+            epilog=epilog_help_string,
+            formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument('--password_file', help='Path to a password file: text file with one line in format <username>:<password>')
+    parser.add_argument('--filelist', required=True, help='Path to a text file each line of which is a path to a video to create task. The videos should be on the file share connected to CVAT, the paths should be relative to the root folder of the CVAT file share. The name of each created task will be equal to the corresponding relative path.')
```

Comment:
It is very strange description. Better to have absolute paths in the file.

Reply:
"The paths should be relative to the root folder of the CVAT file share" --  otherwise CVAT won't understand the paths to the videos (I checked that).
Please, note that the videos must **already** be on CVAT file share, not on the local computer
(in the CVAT dialog "Task Configuration" that appears when I press the button "Create New Task" on CVAT dashboard there is radio button "Source:" with one of the options "Share" ).